cs/v1/CS_v1.0.0.yaml

```diff
@@ -1432,137 +1327,150 @@ components:
-        - modeOfTransportType
+    VesselTransport:
+      title: Vessel Transport
+      type: object
+      description: The vessel transport mode details whne the mode of Transport is Vessel.
       properties:
-        modeOfTransportType:
-          x-stoplight:
-            id: 1xzl152c5xhau
-          enum:
-            - RAIL
-          description: The mode of transport as defined by DCSA. For the Rail Transport mode this needs to be RAIL.
-        railCar:
+        portVisitReference:
           type: string
-          x-stoplight:
-            id: v8mz0l0o5azun
-          description: 'A railcar is a type of railway vehicle that is designed to transport freight or passengers on a railway track. They are also known as rail vehicles, railcars, or rolling stock. Railcars can be powered by an on-board locomotive or they can be pulled by a separate locomotive.'
+          example: NLAMS1234589
           maxLength: 50
-        railService:
+          description: The unique reference that can be used to link different `transportCallReferences` to the same port visit. The reference is provided by the port to uniquely identify a port call.
+        transportCallReference:
+          type: string
+          example: SR11111X-9321483-2107W-NLAMS-ACT-1-1
+          maxLength: 100
+          description: 'The unique reference for a transport call. It’s the vessel operator’s responsibility to provide the Transport Call Reference, other parties are obliged to pick it up and use it. It can take the form of Port Call References as defined in OVS Definitions Document, or alternatively a reference as defined by the vessel operator.'
+        serviceCarriers:
+          type: array
+          items:
+            $ref: '#/components/schemas/ServiceCarrier'
+        universalServiceReference:
+          type: string
+          description: 'A global unique service reference, as per DCSA standard, agreed by VSA partners for the service. The service reference must match the regular expression pattern: SR\d{5}[A-Z]. The letters SR followed by 5 digits, followed by a checksum-character as a capital letter from A to Z.'
+          example: SR12345A
+          pattern: ^SR\d{5}[A-Z]$
+          maxLength: 8
+          minLength: 8
+        universalExportVoyageReference:
+          type: string
+          description: 'A global unique voyage reference for the export Voyage, as per DCSA standard, agreed by VSA partners for the voyage.'
+          pattern: ^\d{2}[0-9A-Z]{2}[NEWSR]$
+          minLength: 5
+          maxLength: 5
+          example: 2103N
+        universalImportVoyageReference:
+          type: string
+          description: 'A global unique voyage reference for the import Voyage, as per DCSA standard, agreed by VSA partners for the voyage.'
+          pattern: ^\d{2}[0-9A-Z]{2}[NEWSR]$
+          minLength: 5
+          maxLength: 5
+          example: 2103N
+        vessel:
+          $ref: '#/components/schemas/Vessel'
+    BargeTransport:
```

Comment:
This object does not have the `carrier`-specific voyages and services? Is that correct?

Reply:
It's inside the serviceCarrier object